includes/exceptions/class-invalid-price-exception.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Class Rest_Request_Exception
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Exceptions;
+
+use Exception;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Exception for throwing errors in REST API controllers (e.g. issues with missing parameters in requests).
+ */
+class Invalid_Price_Exception extends Exception {}
```

Comment:
If we extend `Rest_Exception`, we could log additional properties as well using this.

Reply:
This is not a REST exception, and the one we have in the client (`Rest_Request_Exception`) does not include any additional properties. Considering this is client-side, do you think adding the product ID to the error message could be sufficient?

Reply:
Ah got it confused with the server one. Yeah, adding product id would suffice.

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/4649/commits/b973eb30a00bd00bd8e99443ba8e7b2deec98c18